jme3-core/src/main/java/com/jme3/system/AppSettings.java

```diff
@@ -266,6 +266,7 @@
     public static final String JOAL = "JOAL";
 
     static {
+        defaults.put("CenterWindow", true);
```

Comment:
Perhaps we should supply defaults for WindowXPos and WindowYPos, in case the developer sets CenterWindow but not WindowXPos and WindowYPos?